docs/reference/field-types/relationship-reverse.md

```diff
@@ -1,36 +1,40 @@
+
 ```javascript
 // Configuring the `_pizzas` field in a module's `fields.add` subsection:
 _pizzas: {
   type: 'relationshipReverse',
   withType: 'pizza',
-  reverseOf: '_fabrics'
+  reverseOf: '_toppings'
 }
 ```
 
-We can now see `_pizzas` as a property of each `topping` object that is related to a product.
-
-If desired, we can specify `schema` and `relationshipsField` just as we would for `joinByArray`. Currently these are not automatic in a reverse join and must be fully specified if relationship properties are to be accessed. Most array joins do not have relationship properties and thus do not require reverse access to them.
-
 ## Settings
 
+### Required
+
 |  Property | Type   | Default | Description |
 |-----------|-----------|-----------|-----------|
-|`name` | String | n/a | Sets the name of the field in the database |
-|`label` | String | n/a | Sets the visible label for the field in the UI |
-|`required` | Boolean | `false` | If `true`, the field is mandatory |
 |`type` | String | n/a | Specifies the field type (`relationshipReverse` for this type) |
-|`help` | String | n/a | Help text for the content editor |
-|`htmlHelp` | String | n/a | Help text with support for HTML markup |
-|`withType` | String | n/a | The name of the related type, if it differs from the name of the join. If you do not set `withType`, then the name of the join must match the name of the related type, with a leading `_` added.  |
-|`reverseOf` | String | n/a | Set to the name of the join you are reversing (optional) |
-|`ifOnlyOne` | Boolean | `false` | If `true`, it will only carry out the join if the query that returned the original document returned only one document. This is useful if the joined information is only to be displayed on the `show.html` page of a piece, for instance, and you don't want the performance impact of loading it on the `index.html` page. |
-
-::: tip
-In documents with many joins in play, the `ifOnlyOne` option will avoid running through all the possible joins, and can be used to avoid a heavy performance impact in complex documents.
-:::
+
+### Optional
+
+|  Property | Type   | Default | Description |
+|-----------|-----------|-----------|-----------|
+|`reverseOf` | String | n/a | Set to the name of the related `relationship` field. |
+|`ifOnlyOne` | Boolean | `false` | If `true`, it will only reveal the relationship data if the doc query returned only one document. [See below](#ifonlyone) for more. |
+|`withType` | String | Uses the field name, minus its leading `_` and possible trailing `s` | The name of the related type. |
+
+## `ifOnlyOne`
+
+The `ifOnlyOne` option can provide a performance improvement if the reverse relationship data is only needed when one piece is queried. Setting it to `true` tells Apostrophe not to look for this extra relationship data in contexts where many pieces are queried (e.g., an [index page](/reference/glossary.md#index-page)).
```

Comment:
Setting "it" -> "Setting the option"

"tell Apostrophe not to look for this extra" -> "instructs Apostrophe not to request additional" (tells is actually fine maybe)

"in contexts where many pieces are queried (e.g., an [index page](/reference/glossary.md#index-page))." Is there a way to be a bit more specific here? We give an example of an index page...are there more examples that this is useful in?

Reply:
@abea Is this just duplicate?

Reply:
It's a valid option here as well.